backend/src/main/java/com/bakdata/conquery/commands/ManagerNode.java

```diff
@@ -343,19 +344,17 @@ public void exceptionCaught(IoSession session, Throwable cause) {
 	@Override
 	public void messageReceived(IoSession session, Object message) {
 		ConqueryMDC.setLocation("ManagerNode[" + session.getLocalAddress().toString() + "]");
-		if (message instanceof MessageToManagerNode) {
-			MessageToManagerNode mrm = (MessageToManagerNode) message;
+		if (message instanceof MessageToManagerNode toManagerNode) {
+
 			log.trace("ManagerNode received {} from {}", message.getClass().getSimpleName(), session.getRemoteAddress());
 
-			ReactingJob<MessageToManagerNode, NetworkMessageContext.ManagerNodeNetworkContext>
-					job =
-					new ReactingJob<>(mrm, new NetworkMessageContext.ManagerNodeNetworkContext(
-							new NetworkSession(session),
-							datasetRegistry, config.getCluster().getBackpressure()
-					));
+			Job job = new ReactingJob<>(toManagerNode, new NetworkMessageContext.ManagerNodeNetworkContext(
+					new NetworkSession(session),
+					datasetRegistry, config.getCluster().getBackpressure()
+			));
 
-			if (mrm.isSlowMessage()) {
-				((SlowMessage) mrm).setProgressReporter(job.getProgressReporter());
+			if (toManagerNode.isSlowMessage()) {
+				((SlowMessage) toManagerNode).setProgressReporter(job.getProgressReporter());
 				jobManager.addSlowJob(job);
 			}
```

Comment:
```suggestion
			if (toManagerNode instanceof SlowMessage slowMessage) {
				slowMessage.setProgressReporter(job.getProgressReporter());
				jobManager.addSlowJob(job);
			}
```
Ich hoffe das geht so, das mit den verschachtelten Messages ist nicht so schön

Reply:
also gerne `.isSlowMessage()` abschießen